iree/compiler/Dialect/HAL/Transforms/BUILD

```diff
@@ -34,6 +34,7 @@ cc_library(
         "Passes.cpp",
         "PropagateConstantWorkgroupInfo.cpp",
         "PublicAbiGeneration.cpp",
+        "RepeatDispatches.cpp",
```

Comment:
we should name this more clearly to indicate that it is for microbenchmarking and not a general part of the pipeline - `BenchmarkBatchDispatches` or something.